packages/babel-helper-module-imports/src/import-injector.js

```diff
@@ -89,6 +89,13 @@ export type ImportOptions = {
    *  * false - No particular requirements for context of the access. (Default)
    */
   ensureNoContext: boolean,
+
+  /**
+   * Define whether the import should be loaded before or after the existing imports.
+   * "last" is only allowed inside ECMAScript modules, since it's not possible to
+   * reliably pick the location _after_ require() calls but _before_ other code in CJS.
+   */
+  importOrder: "first" | "last",
```

Comment:
`importOrder: "first"` could be vague when there are multiple `ImportInjector` calls -- only the last call could make it the first import. WDYT rename it to `importPlace: "before" | "after"` or `injectPlace`?

Reply:
I like `importPosition: "before" | "after"`